contracts/abstracts/EIP712.sol

```diff
@@ -6,40 +6,24 @@ abstract contract EIP712 {
     string public constant EIP191_HEADER = "\x19\x01";
 
     // EIP-712 Domain
-    string public constant EIP712_DOMAIN_NAME = "Tokenlon";
-    string public constant EIP712_DOMAIN_VERSION = "v6";
+    bytes32 public constant EIP712_HASHED_NAME = keccak256("Tokenlon");
+    bytes32 public constant EIP712_HASHED_VERSION = keccak256("v6");
+    bytes32 public constant EIP712_TYPE_HASH = keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)");
```

Comment:
I think exposing `EIP712_DOMAIN_NAME` and `EIP712_DOMAIN_VERSION` could be helpful (for third party/users browsing contract from Etherscan?) but exposing the hash of them is probably not too helpful.

Maybe turn hashed value into private values and optionally exposing `EIP712_DOMAIN_NAME` and `EIP712_DOMAIN_VERSION`?

Reply:
Yeah, I think it should be more ideal.

Reply:
Adjusted in 7aef39b